svc-bip-api/src/main/java/gov/va/vro/bip/model/BipPayloadRequest.java

```diff
@@ -1,3 +1,19 @@
 package gov.va.vro.bip.model;
 
-public interface BipPayloadRequest {}
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+import lombok.extern.jackson.Jacksonized;
+
+@Getter
+@Jacksonized
+@SuperBuilder(toBuilder = true)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BipPayloadRequest {
```

Comment:
Semi-nit: should this be abstract?  Will we ever be instantiating a BipPayloadRequest, or only its subtypes?